test/e2e/grpc/inventory_grpc_test.go

```diff
@@ -80,7 +79,6 @@ func waitForServiceToBeReady() error {
 }
 
 func TestInventoryAPIGRPC_livez(t *testing.T) {
-	t.Parallel()
```

Comment:
Removing parallel execution for e2e tests for now, since it is causing issues due to serialized transactions and the retry configuration needs to be updated. We can enable parallel execution after we fix up Inventory a little bit.